recipes-nodes/haproxy/init

```diff
@@ -0,0 +1,56 @@
+#!/bin/sh
+### BEGIN INIT INFO
+# Provides:          haproxy
+# Required-Start:    $remote_fs $syslog $network
+# Required-Stop:     $remote_fs $syslog
+# Default-Start:     2 3 4 5
+# Default-Stop:      0 1 6
+# Short-Description: Start HAProxy service
+### END INIT INFO
+
+PATH=/usr/local/sbin:/usr/local/bin:/sbin:/bin:/usr/sbin:/usr/bin
+NAME=haproxy
+DESC="HAProxy Service"
+LOGFILE=/var/log/containers/haproxy.log
```

Comment:
I would probably make the path /var/log/haproxy.log or we need to adjust our logrotate configuration to also include any subdirectories recursively too. See [here](https://github.com/flashbots/meta-confidential-compute/blob/main/recipes-core/logrotate/files/default-logrotate.conf)

Reply:
Adjusted logrotate config: https://github.com/flashbots/meta-confidential-compute/pull/33

Container logs have to be in the separate directory because Fluentbit uses a non-default `cri` parser for container logs. Fluentbit has 1:1 relation between the path where the logs are stored and the parser applied.